src/octoprint/util/comm.py

```diff
@@ -2730,6 +2766,10 @@ def convert_line(line):
                                 self._logger.info(
                                     "Firmware states that it supports emergency GCODEs to be sent without waiting for an acknowledgement first"
                                 )
+                            elif capability == self.CAPABILITY_EXTENDED_M20 and enabled:
+                                self._logger.info(
```

Comment:
ditto

Reply:
I've left this as INFO level to match the above capability reports, could just remove the whole `elif` section since it's only for logging purposes.